apps/test/unit/gamelab/spritelab/spriteUtilsTest.js

```diff
@@ -0,0 +1,100 @@
+/* global p5 */
+import {expect} from '../../../util/configuredChai';
```

Comment:
use `../../../util/reconfiguredChai` since configuredChai contains a module that is being deprecated.